sentry-android-core/src/main/java/io/sentry/android/core/AndroidContinuousProfiler.java

```diff
@@ -58,6 +58,7 @@ public class AndroidContinuousProfiler
   private @NotNull SentryDate startProfileChunkTimestamp = new SentryNanotimeDate();
   private boolean shouldSample = true;
   private boolean isSampled = false;
```

Comment:
I only see this being used within `startProfileSession()`, can't we make this a local variable then?

Reply:
this is only set when `shouldSample` is true: it's evaluated once per session, then the value is simply reused, so it cannot be a local variable